docs/DeveloperGuide.md

```diff
@@ -390,30 +390,70 @@ and simply import one after the other. As such if the user prefers updating thei
 the command handles that situation too by ignoring duplicate students and simply importing the new updates.
 
 
-### Mailing feature: mail-all
+### Mailing feature
 [<sub><sup>Back to top</sup></sub>](#table-of-contents)
 
+#### Aim of the feature
```

Comment:
Could you place this under Design Considerations, on a point of "Motivation"?

Reply:
Are we not having "Aim of the feature"?